service_container/service_subscribers_locators.rst

```diff
@@ -257,7 +257,7 @@ argument of type ``service_locator``:
         services:
             App\CommandBus:
                 arguments:
-                    !service_locator
+                    !service_locator |
```

Comment:
this looks wrong to me

Reply:
Yeah... But Im not sure what the correct syntax is. https://github.com/symfony/symfony-docs/pull/15250#discussion_r615049079

Reply:
I based my suggestion on https://yaml.org/spec/1.2/spec.html#id2761694 . It seems like you need a multiline indicator if you want multiline tagged values in Yaml

Reply:
I'll revert this change until we know better =)

Reply:
see #15281